llvm/include/llvm/Analysis/CFG.h

```diff
@@ -85,17 +85,27 @@ bool isPotentiallyReachable(
 /// Determine whether there is at least one path from a block in
 /// 'Worklist' to 'StopBB' without passing through any blocks in
 /// 'ExclusionSet', returning true if uncertain.
-///
-/// Determine whether there is a path from at least one block in Worklist to
-/// StopBB within a single function without passing through any of the blocks
-/// in 'ExclusionSet'. Returns false only if we can prove that once any block
-/// in 'Worklist' has been reached then 'StopBB' can not be executed.
-/// Conservatively returns true.
```

Comment:
Do we need to remove this really? I feel better to make it self contained.

Reply:
Restored.